chrome/content/zotero/merge.xhtml

```diff
@@ -34,18 +34,25 @@
 <window xmlns="http://www.mozilla.org/keymaster/gatekeeper/there.is.only.xul"
 		xmlns:html="http://www.w3.org/1999/xhtml"
 		onload="Zotero_Merge_Window.init()"
+		drawintitlebar-platforms="mac"
+		width="830" height="500"
```

Comment:
Don't `width` and `height` not work on `window` anymore?

Reply:
It works somehow. Otherwise, the window will only have a default small size like this:

<img width="520" alt="image" src="https://github.com/zotero/zotero/assets/33902321/7b7ec4cb-afd6-4e67-b8af-047daef6527b">

Reply:
I think it's no longer working on most of the XUL elements, but maybe the window is a special case where you need to decide its size before loading any CSS, and that's why we can use `width`/`height` here.

Reply:
Hmm, maybe still works on modal dialogs somehow? Still might be better to use inline `min-width`, since this seems likely to break later?

Reply:
done